### DISCUSSION TITLE
Borders opacity global variable

### DISCUSSION THREAD

**Discussion by Vemq** [vuetifyjs/vuetify#20581] [Q&A]
How can I set border opacity globaly without affecting other components?

The default value for  variable `--v-border-opacity` is 0.12, but I need the default value of 1.0.

Changing `--v-border-opacity` variable breaks other components. For example it is used in VBtn overlay, VProgressLinear background,  VSkeletonLoader background, ect.: 
https://github.com/search?q=repo%3Avuetifyjs%2Fvuetify%20--v-border-opacity&type=code

I can remove `--v-border-opacity` from `$border-color-root` variable, but then I can't use `border-opacity-` classes. It's not impotatn, but not good anyway.
Using the `border-opacity-100` class everywhere I want borders is too redundant.

And why is the `--v-border-opacity` variable used in inappropriate places? Not only for borders, as in the above examples.

**Comment by karwan01** [comment]
Depending on your setup, you could also extend Vuetify's theme properties via the plugin options in your main.js file:

```vue
import { createVuetify } from 'vuetify';

export default createVuetify({
  theme: {
    themes: {
      light: {
        variables: {
          'border-opacity': '1', // Custom variable to apply in borders only
        },
      },
    },
  },
});

**Reply by Vemq** [reply]
> Changing --v-border-opacity variable breaks other components. For example it is used in VBtn overlay, VProgressLinear background, VSkeletonLoader background, ect.:
> https://github.com/search?q=repo%3Avuetifyjs%2Fvuetify%20--v-border-opacity&type=code

**Reply by karwan01** [reply]
no i mean you can create your own custom variables in vuetify
